install/kubernetes/cilium/templates/clustermesh-apiserver/service.yaml

```diff
@@ -41,4 +41,8 @@ spec:
   {{- if or (eq .Values.clustermesh.apiserver.service.enableSessionAffinity "Always") (and (eq .Values.clustermesh.apiserver.service.enableSessionAffinity "HAOnly") (gt (int .Values.clustermesh.apiserver.replicas) 1)) }}
   sessionAffinity: ClientIP
   {{- end }}
+  {{- if and (eq "LoadBalancer" .Values.clustermesh.apiserver.service.type) .Values.clustermesh.apiserver.service.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
+    {{- toYaml .Values.clustermesh.apiserver.service.loadBalancerSourceRanges | nindent 4 }}
+  {{- end }}
```

Comment:
Nit. I'd suggest moving this setting close to the other LoadBalancer related ones, for consistency.

Reply:
Thank you for this. I've moved it up to be just after `LoadBalancerIp` so the LoadBalancer attributes are now close to one another + sorted alphabetically.